gradle.properties

```diff
@@ -33,7 +33,7 @@ githubRepo = summoningrituals
 # Dependencies
 jeiVersion = 11.3.0.271
 jeiVersionRange = [11.0,12.0)
-kubeVersion = 1902.6.0-build.98
+kubeVersion = 1902.6.0-build.119
 kubeVersionRange = [1902.6,1902.7)
```

Comment:
Please adjust the minimum version from the version range to reflect the minimum required KubeJS version in order for this change to work.